includes/admin.php

```diff
@@ -215,3 +215,47 @@ function pmprogroupacct_plugin_row_meta($links, $file) {
 	return $links;
 }
 add_filter('plugin_row_meta', 'pmprogroupacct_plugin_row_meta', 10, 2);
+
+
+/**
+ * Import group account member associations when using the Import Users From CSV plugin.
+ *
+ * @param WP_User $user The user object that was imported.
+ * @param int $membership_id The membership level ID that was imported.
+ * @param MemberOrder|null $order The order object that was created during import. Null if no order is created.
+ * @since TBD
+ */
+function pmprogroupacct_pmproiucsv_post_user_import( $user, $membership_id, $order ) {
+	global $wpdb;
+
+	$group_level_settings =  pmprogroupacct_get_settings_for_level( $membership_id );
+	$parent_group = PMProGroupAcct_Group::get_group_by_parent_user_id_and_parent_level_id( $user->ID, $membership_id );
+	$group_id = $user->pmprogroupacct_group_id; // Child accounts would pass through the Group ID.
+	$seats = ! empty( $user->pmprogroupacct_group_total_seats ) ? intval( $user->pmprogroupacct_group_total_seats ) : $group_level_settings['max_seats'];
```

Comment:
Do we always want to default to the max seats for a group, or if the number of seats is not specified, do we instead just not want to alter group seat data?